Resources/Computing/PoolComputingElement.py

```diff
@@ -49,13 +64,19 @@ def _addCEConfigDefaults( self ):
     # First assure that any global parameters are loaded
     ComputingElement._addCEConfigDefaults( self )
 
-  def getCoresInUse( self ):
+  def _reset( self ):
+
+    self.processors = self.ceParameters.get( 'NumberOfProcessors', self.processors )
```

Comment:
I think all these should be "casted" to their expected type because the ceParameters are just strings.